assemblies/core/lib/pom.xml

```diff
@@ -628,39 +625,6 @@
                 </exclusion>
             </exclusions>
         </dependency>
-        <dependency>
```

Comment:
By removing jackson from here aren't we removing it from data-integration/lib? Doesn't this break non OSGi plugins that assume that this lib is available?
@hbfernandes ^

Reply:
In the last test stack build we have `jackson-annotations-2.9.5.jar`, `jackson-databind-2.9.5.jar` and `jackson-core-2.9.5.jar` in _data-integration/lib_. We have these three declared as dependencies of **kettle-engine**. Do we also want to make these as explicit dependencies of **assemblies/core/lib**? @graimundo @hbfernandes

Reply:
Same comment as: https://github.com/pentaho/pentaho-platform/pull/4161#discussion_r198817326

@pentaho-lmartins I think that for the scope of this story it is ok to remove the explicit dependencies and let them come in transitively. That means we are assuming that the lib contains the necessary libs for the Pentaho code base and that we are not putting anything there just for the sake of third parties. So lets go with this solution to unblock the work and create that followup thread that we talked about. If necessary, as a result of that discussion we can re add the dependencies explicitly later on.